init.sql

```diff
@@ -7,23 +7,56 @@ CREATE ROLE readonly;
 GRANT USAGE ON SCHEMA public TO readonly;
 GRANT SELECT ON ALL TABLES IN SCHEMA public TO readonly;
 
-CREATE USER mainnet with password 'password';
-GRANT ALL PRIVILEGES ON ALL TABLES IN SCHEMA public TO mainnet;
-GRANT ALL PRIVILEGES ON ALL SEQUENCES IN SCHEMA public TO mainnet;
-
-CREATE TABLE daily_statistics
-(
-    computed_for_timestamp     numeric(20, 0) NOT NULL,
-    transactions_count         numeric(20, 0) NOT NULL,
-    teragas_used               numeric(20, 0) NOT NULL,
-    deposit_amount             numeric(45, 0) NOT NULL,
-    new_accounts_count         numeric(20, 0) NOT NULL,
-    deleted_accounts_count     numeric(20, 0) NOT NULL,
-    active_accounts_count      numeric(20, 0) NOT NULL,
-    new_contracts_count        numeric(20, 0) NOT NULL,
-    new_unique_contracts_count numeric(20, 0) NOT NULL,
-    active_contracts_count     numeric(20, 0) NOT NULL
-);
-
-ALTER TABLE ONLY daily_statistics
-    ADD CONSTRAINT daily_statistics_pkey PRIMARY KEY (computed_for_timestamp);
+CREATE TABLE daily_transactions_count
+(
+    daily_timestamp    numeric(20, 0) PRIMARY KEY,
+    transactions_count numeric(20, 0) NOT NULL
+);
+
+CREATE TABLE daily_teragas_used
+(
+    daily_timestamp numeric(20, 0) PRIMARY KEY,
+    teragas_used    numeric(20, 0) NOT NULL
```

Comment:
Let's have all the ranges documented, so it is obvious that the type cannot overflow.

I would also like you to investigate how PostgreSQL will handle the values if we sum them up:

```sql
SELECT SUM(gas_used) FROM daily_gas_used
```

Reply:
It gives `ERROR:  numeric field overflow`
Or it gives correct value if we use cast (`SUM(gas_used)::numeric(100,0)`)
I'll think about ranges carefully